web/src/main/java/org/cbioportal/web/util/RequestWrapper.java

```diff
@@ -0,0 +1,47 @@
+package org.cbioportal.web.util;
```

Comment:
@sheridancbio this isn't used right?

Reply:
Maybe not .. maybe this is left in from earlier attempts at this?

Reply:
file removed and tests still pass